cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -46,10 +46,11 @@
             "HANCESTRO:0575",
             "HsapDv:0000003",
             "donor_1",
+            "nucleus",
         ],
         [
             "CL:0000192",
-            "EFO:0010183",
+            "EFO:0009918",
```

Comment:
changed because previous example was not a valid EFO for suspension_type, would add warnings to all the test outputs